### DISCUSSION TITLE
stabilityDays at 2 and internalChecksFilter at flexible

### DISCUSSION THREAD

**Discussion by mcantinqc** [renovatebot/renovate#18736] [Request Help] [answered]
Hi,

I'm now sure about how to use the internalChecksFilter at flexible. 

We can read on the documentation : 
> flexible: Similar to strict, but in the case where all versions are pending then PR will be created with the highest pending version

From my understanding, like in the strict mode PR isn't supposed to be open before the stabilityDays requirement is met, the change compared with the strict mode is when the PR will be opening that will use the latest version and not the "stable" version.

But the actual behaviour is the PR is open with the check "renovate/stability-days" like in the "none" mode.

I’m right and this is a bug?

**Comment by mcantinqc** [comment]
The option "prCreation" option at "not-pending" do what I suppose "internalChecksFilter" supposed to do.

**Comment by rarkins** [comment, marked as answer]
It's not a bug. internalChecksFilter=flexible behaves like internalChecksFilter=none if all releases don't pass stabilityDays
